keepassxc-browser/content/passkeys.js

```diff
@@ -34,7 +34,12 @@ const createAttestationResponse = function(publicKey) {
         getTransports: () => [ 'internal' ]
     };
 
-    return Object.setPrototypeOf(response, AuthenticatorAttestationResponse.prototype);
+    // Prevent Illegal invocation error
+    const responseWithProto = Object.setPrototypeOf(response, AuthenticatorAttestationResponse.prototype);
+    responseWithProto.getPublicKey = () => null;
```

Comment:
Why not define these directly to the `response` above?

Reply:
We can merge it